backend/app/Listeners/SendAccountConnectedNoDisplayNotification.php

```
@@ -0,0 +1,23 @@
+class SendAccountConnectedNoDisplayNotification
+{
+    public function handle(AccountConnectedNoDisplay $event): void
+    {
+        $webhookUrl = config('settings.account_connected_no_display_webhook_url');
+        if (!$webhookUrl) {
+            return;
+        }
+
+        Http::post($webhookUrl, [
+            'event' => 'account_connected_no_display',
+            'user' => UserWebhookData::from($event->user),
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Data\UserWebhookData::from().


(staticMethod.notFound)